runatlantis.io/docs/policy-checking.md

```diff
@@ -49,9 +49,36 @@ policies:
       source: local
 ```
 
-`name` - A name of your policy set.
-`path` - Path to a policies directory.
-`source` - Tells atlantis where to fetch the policies from. Currently you can only host policies locally by using `local`.
+- `name` - A name of your policy set.
+- `path` - Path to a policies directory. *Note: replace `<CODE_DIRECTORY>` with absolute dir path to conftest policy/policies.*
+- `source` - Tells atlantis where to fetch the policies from. Currently you can only host policies locally by using `local`.
+
+By default conftest is configured to only run the `main` package. If you wish to run specific/multiple policies consider passing `--namespace` or `--all-namespaces` to conftest with [`extra_args`](https://www.runatlantis.io/docs/custom-workflows.html#adding-extra-arguments-to-terraform-commands) via a custom workflow as shown in the below example.
+
+Example Server Side Repo configuration using `--all-namespaces` and a local src dir.
+
+```
+repos:
+  - id: github.com/myorg/example-repo
+policies
+  owners:
+    users:
+      - example-dev
+    policy_sets:
```

Comment:
This one is indented one to much as it's not a part of the owners object?

Reply:
Correct. I'll put a fix pr together for both.